### PR TITLE
LLD: Update Sentry + sampling 100% + more errors filtering

### PR DESCRIPTION
**Pull request by gre**
### 📝 Description

- Update Sentry to @sentry/electron 4.0.0
- error sampling from 50% to 100%
- ignore more cases of errors

### ❓ Context

- **Impacted projects**: `LLD` 
- **Linked resource(s)**: live-3508 

### ✅ Checklist

- [ ] **Test coverage** to test, we need to manual test with this: https://github.com/LedgerHQ/ledger-live/wiki/Manually-Test-Sentry-integration
- [x] **Atomic delivery** 
- [x] **No breaking changes** 

### 📸 Demo

I have manually tested this build on Mac OS:

Screenshot 2022-08-25 at 16 24 04

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

**Comment by gre** [comment]
honestly I have no idea why i needed to regen pods here. maybe someone have an idea?

File: pnpm-lock.yaml [hunk 1]
```diff
@@ -54781,13 +55375,5 @@ packages:
     peerDependencies:
       react-native: '>=0.60 <1.0.0'
     dependencies:
-      react-native: 0.68.2_shn3lkepuadv4od4dudas6ijbq
+      react-native: 0.68.2_qiqqwmyv63yhnuboofexv3s7x4
     dev: false
-
-patchedDependencies:
```
Comment (elbywan) [inline pnpm-lock.yaml:54787]: not sure this is a good thing to have patchedDepencies removed here 🤔

**Comment by gre** [comment]
@valpinkman / @elbywan  : i'm having difficulties understanding why updating only LLD part impacts LLM part. this is a bit a problem because i really only want to update Sentry on LLD. i have a feeling this is due to some sort of dependencies sharing in this mono repo system?

**Comment by elbywan** [comment]
@gre I think it comes from this: [`patched: true`](https://github.com/LedgerHQ/ledger-live/pull/1058/files#diff-32824c984905bb02bc7ffcef96a77addd1f1602cff71a11fbbfdd7f53ee026bbR45784) thing in the lockfile.

Maybe someone pushed a lockfile on dev with a previous pnpm version or something similar.
Worst case, there is 0 impact as the only thing which changed is the hash in the path of the native modules.

**Comment by gre** [comment]
i'll regen then

**Review by haammar-ledger — APPROVED** [review]

**Review by Justkant — APPROVED** [review]
LGTM but still strange that we need to update the podfile for this, I'll test locally with and without the changes on the podfile.lock

**Pull request merged**